### PR TITLE
publish changelog to forum

### PR DESCRIPTION
**Pull request by zuchka**
resolves https://github.com/grafana/grafana-community-support-squad/issues/50

### CODE REVIEW

File: update-changelog/index.ts [hunk 1]
```diff
@@ -119,6 +121,29 @@ class UpdateChangelog extends Action {
 				labels: ['backport ' + versionMajorBranch, 'no-changelog'],
 			})
 		}
+
+		// publish changelog to community.grafana.com
+		const forumData = {
```
Comment (tolzhabayev) [inline update-changelog/index.ts:126]: nit: `forumData` can be moved into the `if(apiKey)`

File: update-changelog/index.ts [hunk 2]
```diff
@@ -119,6 +121,29 @@ class UpdateChangelog extends Action {
...
+			title: `Changelog: Updates in Grafana ${version}`,
+			raw: `${changelog}`,
+			category: 9,
+		}
+		const apiKey = getInput('grafanabotForumKey')
+
+		if (apiKey) {
+			axios({
```
Comment (tolzhabayev) [inline update-changelog/index.ts:134]: nit: since we usually `await`, I would rather `await` with `try/catch` on this `axios` call too.

**Review by tolzhabayev — APPROVED** [review]
Looks good in general, just some small nits that you can ignore.

**Review by natellium — APPROVED** [review]

File: update-changelog/index.ts (hunk 1 above)
Comment (zuchka) [reply update-changelog/index.ts:126]: moved. ty!

File: update-changelog/index.ts (hunk 2 above)
Comment (zuchka) [reply update-changelog/index.ts:134]: ok, wrapped in try/catch and awaiting the axios call

**Pull request merged**